docs/book/src/developer/architecture/controllers/metadata-propagation.md

```diff
@@ -1,25 +1,52 @@
 # Metadata propagation
 Cluster API controllers implement consistent metadata (labels & annotations) propagation across the core API resources.
 This behaviour tries to be consistent with Kubernetes apps/v1 Deployment and ReplicaSet.
-New providers should behave accordingly fitting within the following pattern: 
+New providers should behave accordingly fitting within the following pattern:
+
+<!--The original of the image can be found here: https://docs.google.com/presentation/d/1kotJg9_ql3znlDirAodQYr_CysPSiZOKqknHqXomDyw/edit#slide=id.g221b54432a5_0_214-->
+![](../../../images/metadata-propagation.jpg)
```

Comment:
last nit: can we export this single slide and check it in next to the picture?

Would just prefer if the future edit'able of this jpg in our book doesn't depend on this presentation staying online

Reply:
Ideally the same for the png's linked in our proposal

Reply:
Added the PPT.

>Ideally the same for the png's linked in our proposal (if all 3 of them are also screenshots from the presentation)

The corresponding slides are already included in the repo.